tasks/webstorm.js

```diff
@@ -357,6 +357,8 @@ function executablePath() {
  * @returns {string} A true concatentated path where found, else undefined
  */
 function maximisePath() {
+
```

Comment:
This method is resolving a path that contains some combination of strings and `Regexp`. Previous algorithm performed a directory list for both types of element, even though it was only needed for the `Regexp` element.

There is an obscure bug on windows where a shortcut file (found in `AppData` on some windows machines) is crashing `fs` operations. The new algorithm should not be any worse and should hopefully fix the problem.

Tested on Mac and Windows, but not yet on a machine that exhibits the bug.